ouroboros-consensus/src/Ouroboros/Consensus/Util/EarlyExit.hs

```diff
@@ -191,6 +191,7 @@ instance MonadThread m => MonadThread (WithEarlyExit m) where
 
   myThreadId  = lift    myThreadId
   labelThread = lift .: labelThread
+  threadStatus t = lift (threadStatus t)
```

Comment:
Please use `.:` like the line just above it.

Reply:
I swear I tried this and it didn't work, but apparently it does :shrug:

Reply:
It didn't when I tried https://github.com/input-output-hk/ouroboros-network/actions/runs/3382896942/jobs/5618282705#step:17:109

Reply:
ah right, you reverted. I also gave up trying to work out why and just wrote the obviously correct but dumber one

Reply:
It should work with `.` instead of `.:`

Reply:
I thought I tried that, but apparently not!  Thanks!